operators/cmd/tenant-operator/main.go

```diff
@@ -88,6 +91,9 @@ func main() {
 	flag.StringVar(&ncURL, "nc-url", "", "The base URL for the nextcloud actor.")
 	flag.StringVar(&ncTnOpUser, "nc-tenant-operator-user", "", "The username of the acting account for nextcloud.")
 	flag.StringVar(&ncTnOpPsw, "nc-tenant-operator-psw", "", "The password of the acting account for nextcloud.")
+	flag.DurationVar(&requeueTimeMinimum, "tenant-operator-rq-time-min", 4*time.Hour, "Minimum time elapsed before requeue of controller.")
+	flag.DurationVar(&requeueTimeMaximum, "tenant-operator-rq-time-max", 8*time.Hour, "Maximum time elapsed before requeue of controller.")
+	flag.DurationVar(&tenantNSKeepAlive, "tenant-operator-ns-keep-alive", time.Hour, "Time elapsed after last login of tenant during which the tenant namespace should be kept alive: after this period, the controller will attempt to delete the tenant personal namespace.")
```

Comment:
```suggestion
	flag.DurationVar(&tenantNSKeepAlive, "tenant-operator-ns-keep-alive", 7*time.Days, "Time elapsed after last login of tenant during which the tenant namespace should be kept alive: after this period, the controller will attempt to delete the tenant personal namespace.")
```

Reply:
Fixed in [e17fe5b](https://github.com/netgroup-polito/CrownLabs/pull/821/commits/e17fe5b6dd9c3a0e1519a8924264099dfd8e21e6)